packages/bruno-electron/src/store/preferences.js

```diff
@@ -17,6 +17,7 @@ const defaultPreferences = {
     },
     storeCookies: true,
     sendCookies: true,
+    saveRequestOnRun: true,
```

Comment:
I'm not sure, but might be worth considering: maybe the default value should be false to keep the same behavior for current users?